ocean/src/social.rs

```diff
@@ -1,10 +1,15 @@
+use std::collections::HashMap;
+
 #[derive(Debug)]
 pub struct SocialNetwork {
-    //TO-DO: Add appropriate field
+    friendships: HashMap<String, Vec<String>>
```

Comment:
I think using there are better choices like a Set maybe so I don't know why you chose this.